index.js

```diff
@@ -104,7 +104,7 @@ function parseVideoName(filePath, options)
                 meta.season = parseInt(seasonMatch[1], 10);
             
             /* TODO: consider the case when a hyphen is used for multiple episodes ; e.g. e1-3 */
-            var episodeMatch = x.match(/E(\d{2})/ig);
+            var episodeMatch = x.match(/(?<=\W|\d)E(\d{2})/ig);
```

Comment:
It basically expects there's a number or a non-alphanumeric char to be prefixed to `E`. Like `1E` as in `S01E01` or `S01 E01` or `S01-E01`